index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Version</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="header">
+      <a href="#"><h4>PEDRO MARQUES</h4></a>
+      <nav><img src="images/menu.svg" alt="Menu"></nav>
+    </header>
+    <section class="grid main">
+      <h1>Hey there. I’m Pedro <span> I’m a software developer</span></h1>
+      <p>
+         I can help you build a product , feature or website Look through
+         some of my work and experience! If you like what you see and 
+         have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <ul>
+        <li><a href=""><img src="images/twitter.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/Linkedin-icon.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/medium.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/github.svg" alt="social-icons"></a></li>
+        <li><a href=""><img src="images/angellist.svg" alt="social-icons"></a></li>
+      </ul>
+        <ul>
+          <li><a href=""><img src="images/Disabled.svg" alt="disabled"></a></li>
+        </ul>
+          <h1>My Recent <br> Works</h1>
+        <ul>
+          <li><a href=""><img src="images/Indicator.svg" alt="indicator"></a></li>
+        </ul>
+    </section>
+    <section class="grid works">
+      <h1>Multi-Post Stories <br> Gain+Glory</h1>
+      <ul>
+        <li>Ruby on rails</li>
+        <li>css</li>
+        <li>JavaScript</li>
+        <li>html</li>
+      </ul>
+      <p>See Project<p>
+    </section>
+    <section class="grid aboutme">
+      <h2>About me</h2>
+      <p>
+        I can help you build a product , feature or website Look through
+        some of my work and experience! If you like what you see and 
+        have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <h1>Get My Resume</h1>
+      </section>
+    <section class="grid languages">
+      <h2>Languages</h2>
+      <ul>
+        <li>Ruby on rails</li>
+        <li>css</li>
+        <li>JavaScript</li>
+        <li>html</li>
+      </ul>
+    </section>
+    <section class="grid framework">
+      <h2>Framework</h2>
+      <ul>
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>Rspec</li>
+        <li>Capybara</li>
+        <li>Selenium</li>
+      </ul>
+    </section>
+    <section class="grid skills">
+      <h2>Skills</h2>
+      <ul>
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>          
+      </ul>
```

Comment:
- [ ] Kindly format the skills section correctly as seen in the original/figma design

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/159134672-3ad488fc-c6d7-4b5d-9827-4a893736aea9.png) | ![image](https://user-images.githubusercontent.com/4247740/159134687-6cbec5d0-2e1a-48c7-a087-ec086184b91e.png) |